lib/classes/CLI.js

```diff
@@ -65,8 +65,14 @@ class CLI {
 
   generateMainHelp() {
     this.consoleLog('');
+    this.consoleLog(chalk.yellow('The Serverless Framework - https://serverless.com'));
+    this.consoleLog('');
+
+    this.consoleLog(chalk.yellow('Version:'));
+    this.consoleLog(version);
+    this.consoleLog('');
 
-    this.consoleLog(chalk.yellow.underline('Commands'));
+    this.consoleLog(chalk.yellow('Commands:'));
```

Comment:
Please post screenshot how it looks in your terminal (extra points with details about the terminal config you use)